src/test/java/LoginTests.java

```diff
@@ -0,0 +1,70 @@
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+
+public class LoginTests extends BaseTest {
+    @Test
+    public void LoginWrongCredentialTest() throws InterruptedException {
+
+        // openLoginPage();
+        //enterEmail("europlatinum@mail.ru");
+        //enterPassword("Newjob2!");
+        driver.get("https://kwidos.com/auth/login");
+        driver.findElement(By.cssSelector("#email")).sendKeys("europlatinum@mail.ru");
+        Thread.sleep(4000);
+        driver.findElement(By.cssSelector("#password")).sendKeys("Newjob2!");
+        driver.findElement(By.cssSelector("[type='submit']")).click();
+        String actualText = getErrorText(".alert.alert-danger");
+        Assert.assertEquals(actualText, "Username or password is incorrect");
+    }
+    @Test
+    public void LoginEmptyEmailTest() throws InterruptedException {
+        // openLoginPage();
+        // enterEmail("");
+        driver.get("https://kwidos.com/auth/login");
+        driver.findElement(By.cssSelector("#email")).sendKeys(" ");
```

Comment:
Please move all locators to class variables because they are used in other methods.